app/mailers/invitation_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class InvitationMailer < ApplicationMailer
+  def invitation_email
+    @user = params[:user]
+    @invitee_handle = params[:invitee_handle]
+    @url = "https://brownfield-of-dreams-sb-rp.herokuapp.com/users/#{@user.id}/activate"
```

Comment:
I don't think this is where we want to send a new user (since it takes us to their friend's dashboard if I'm not mistaken). Should we send them to the registration page instead?